chips/nrf52/src/uicr.rs

```diff
@@ -188,4 +191,28 @@ impl Uicr {
     pub fn set_ap_protect(&self) {
         self.registers.approtect.write(ApProtect::PALL::ENABLED);
     }
+
+    /// Disable the access port protection in the UICR register. This is stored
+    /// in flash and is persistent. This behavior can also be accomplished
+    /// outside of tock by running `nrfjprog --recover`.
+    ///
+    /// The `hardware_only` flag indicates whether
+    pub fn disable_ap_protect(&self) {
```

Comment:
It doesn't look like you ever call this. Did you mean to?

Reply:
Added.